Analysis/HiggsTauTau/src/HTTSequence.cc

```diff
@@ -1441,7 +1528,7 @@ if((strategy_type == strategy::fall15 || strategy_type == strategy::mssmspring16
         .set_mt_xtrig_mc(new TH2D(mt_xtrig_mc)).set_mt_xtrig_data(new TH2D(mt_xtrig_data))
         .set_mt_conditional_mc(new TH2D(mt_conditional_mc)).set_mt_conditional_data(new TH2D(mt_conditional_data));
     }else{
-        httWeights.set_scalefactor_file("input/scale_factors/htt_scalefactors_v5.root");
+        httWeights.set_scalefactor_file("input/scale_factors/htt_scalefactors_v4.root");
```

Comment:
not sure you want to do this! We are using v5 now right? This should be taken from the setting on master